### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by iamchristianani**
I and my partner were able to do the following:
- We added a new JS file and connected it to our index file.
- We added the navigation menu for the mobile view and linked them to their respective sections on the page.
- We styled the new navigation menu in our stylesheet while making use of flexbox.
- We added some manipulations to our page using Javascript by
  - making the mobile menu open and close through the hamburger and the X icon.
  - closing the menu when a navigation link is being clicked.
- We made use of Linters to fix the CSS and JavaScript errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,25 +1,39 @@
...
                 <ul class="desktop-menu">
-                    <li>Portfolio</li>
-                    <li>About</li>
-                    <li>Contact</li>
+                    <li><a href="#work-section">Portfolio</a></li>
+                    <li><a href="#about-section">About</a></li>
+                    <li><a href="#contact-section">Contact</a></li>
                 </ul>
             </div>
         </nav>
```
Comment (HAMMAS-SALEEM) [inline index.html:40]: Capture

- [ ] You are missing that section in your project menu which has been shown in the Figma image above 👍 Kindly add that section in your mobile menu to make it similar to the Figma template 😊

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @iamchristianani, @yuvenalmash :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Github flow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css [hunk 2]
```diff
@@ -50,8 +56,8 @@ a {
   display: none;
 }
 
-.menu-btn {
-  margin-top: 5px;
+.menu-btn .fa-bars {
+  font-size: 25px;
 }
```
Comment (christianbiring1) [inline css/style.css:61]: [OPTIONAL]
- [ ] Kindly make sure on the desktop we are not seeing the navbar when the page refreshes. The menu is supposed to be visible only on the mobile version. Kindly correct that... :warning: :warning:  :warning: 

Your design :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @iamchristianani , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
# REQUEST CHANGES :recycle: 
soclose


 ## Highlights
- [ ] All linters are green :heavy_check_mark: 
- [ ] Descri[tive PR `title` and `summary`

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Santiago220991 — APPROVED** [review]
Hi @iamchristianani @yuvenalmash, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
:heavy_check_mark: No linter errors.
:heavy_check_mark: Applied changes suggested by the last reviewer.
:heavy_check_mark: Followed GitHub flow.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**